README.md

```diff
@@ -69,18 +86,68 @@ It is also possible to bind to multiple topics in a single call:
+| `retryDelay`                     | `int`      | `50`         | `kafkaRetry` | The amount of time in milliseconds that is waited before retrying to process a previously failed message.                                                                                                             |
+| `messageFailureMaxTime`          | `int`      | `null`       | `kafkaRetry` | The maximum amount of time in milliseconds that the retry process must last before retry of the message process is aborted.                                                                                           |
+| `messageDelayExponential`        | `int`      | `2`          | `kafkaRetry` | The exponential used to increase the delay between retries of a previously failed message processing.                                                                                                                 |
+| `retryPersistenceDir`            | `str`      | `data`       | `kafkaRetry` | Kafka retry consumer persistent directory that will store files for message retries.                                                                                                                                  |
 
-| Name                                     | Type  | Default            | Description                                                                                                                                      |
-| ---------------------------------------- | ----- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------ |
-| **BUS_ADAPTER**                          | `str` | `kafka`            | The name of the bus adapter adapter to be used (eg: `kafka`, `kafkaRetry`).                                                                      |
-| **KAFKA_HOSTS**                          | `str` | `localhost:9092`   | The hostname and port of the Kafka nodes to be used, separated by a `,` (comma).                                                                 |
-| **KAFKA_CLIENT_ID**                      | `str` | `ripe-kafka`       | Kafka client ID, uniquely identifies a connection to a Kafka broker (more [here](https://kafka.js.org/docs/1.13.0/configuration)).               |
-| **KAFKA_CONSUMER_GROUP_ID**              | `str` | `${RANDOM_STRING}` | Kafka consumer group ID, identifies a consumer and is unique for a given Kafka broker (more [here](https://kafka.js.org/docs/1.13.0/consuming)). |
-| **KAFKA_CONSUMER_RETRY_PERSISTENCE_DIR** | `str` | `data`             | Kafka consumer persistent directory that will store files for message retries.                                                                   |
-| **KAFKA_RETRIES**                        | `int` | `5`                | Kafka max number of retries to be used in the sending of a message.                                                                              |
+## Configuration
```

Comment:
For Global Configuration and Kafka configuration, I would maybe split into Global and then for Kafka (like in [core](https://github.com/ripe-tech/ripe-core/)). In this case, it's only `BUS_ADAPTER` but will allow for different tables for different adapters and potentially new global variables.